server/planning/commands/purge_expired_locks_test.py

```diff
@@ -24,102 +26,138 @@
+    app_config = {
+        **TestCase.app_config.copy(),
+    }
+
     async def asyncSetUp(self) -> None:
         await super().asyncSetUp()
-        self.app.data.insert(
-            "events",
-            [
-                {
-                    "_id": "active_event_1",
-                    "dates": {"start": now, "end": now + timedelta(days=1)},
-                    "lock_user": "user1",
-                    "lock_session": "session1",
-                    "lock_time": now - timedelta(hours=23),
-                    "lock_action": "edit",
-                },
-                {
-                    "_id": "expired_event_1",
-                    "dates": {"start": now, "end": now + timedelta(days=1)},
-                    "lock_user": "user2",
-                    "lock_session": "session2",
-                    "lock_time": now - timedelta(hours=25),
-                    "lock_action": "edit",
-                },
-            ],
-        )
-        self.app.data.insert(
-            "planning",
-            [
-                {
-                    "_id": "active_plan_1",
-                    "planning_date": now,
-                    "lock_user": "user3",
-                    "lock_session": "session3",
-                    "lock_time": now - timedelta(hours=23),
-                    "lock_action": "edit",
-                },
-                {
-                    "_id": "expired_plan_1",
-                    "planning_date": now,
-                    "lock_user": "user4",
-                    "lock_session": "session4",
-                    "lock_time": now - timedelta(hours=25),
-                    "lock_action": "edit",
-                },
-            ],
-        )
-        self.app.data.insert(
-            "assignments",
-            [
-                {
-                    "_id": assignment_1_id,
-                    "lock_user": "user5",
-                    "lock_session": "session5",
-                    "lock_time": now - timedelta(hours=23),
-                    "lock_action": "edit",
-                },
-                {
-                    "_id": assignment_2_id,
-                    "lock_user": "user6",
-                    "lock_session": "session6",
-                    "lock_time": now - timedelta(hours=25),
-                    "lock_action": "edit",
-                },
-            ],
-        )
-        self.assertLockState(
-            [
-                ("events", "active_event_1", True),
-                ("events", "expired_event_1", True),
-                ("planning", "active_plan_1", True),
-                ("planning", "expired_plan_1", True),
-                ("assignments", assignment_1_id, True),
-                ("assignments", assignment_2_id, True),
-            ]
-        )
-
-    def test_invalid_resource(self):
-        with self.assertRaises(ValueError):
-            PurgeExpiredLocks().run("blah")
+        self.app_config.update({"MODULES": ["planning.module"]})
```

Comment:
This is already defined in the parent class' `app_config`, so this line can be removed